samples/sensor/max30101/src/main.c

```diff
@@ -24,10 +24,10 @@ void main(void)
 
 	while (1) {
 		sensor_sample_fetch(dev);
-		sensor_channel_get(dev, SENSOR_CHAN_GREEN, &green);
+		sensor_channel_get(dev, SENSOR_CHAN_RED, &red);
 
-		/* Print green LED data*/
-		printf("GREEN=%d\n", green.val1);
+		/* Print red LED data*/
+		printf("RED=%d\n", red.val1);
 
 		k_sleep(K_MSEC(20));
 	}
```

Comment:
I think this commit should be squashed with the driver one. This one alone (w/o the multi-instance driver) has no sense. The commit is an atomic piece of change that spans across multiple parts in the projects (driver, sample, dts bindings, test, ...)